src/java.base/share/classes/jdk/internal/misc/ThreadTracker.java

```diff
@@ -42,7 +42,7 @@ public class ThreadTracker {
     private record ThreadRef(Thread thread) {
         @Override
         public int hashCode() {
-            return Long.hashCode(thread.threadId());
+            return Long.hashCode(thread.getId());
         }
```

Comment:
- The method `threadId()` does not exist in jdk17u-dev code base yet, but exist in current latest https://github.com/openjdk/jdk/blame/4018b2b19629ddb8cd7a56e064dfef371f23e5fa/src/java.base/share/classes/java/lang/Thread.java#L2565
- The `getId()` method logic is exact the same as `threadId()` method

```java
    @Deprecated(since="19")
    public long getId() {
        return threadId();
    }

    public final long threadId() {
        return tid;
    }
```